droneml_dialog.py

```diff
@@ -1,16 +1,18 @@
-from qgis.PyQt import QtWidgets
+from pathlib import Path
+from qgis.PyQt import QtWidgets, QtCore
 from qgis.core import QgsRasterLayer, QgsVectorLayer, QgsProject
-import os
-import shapely
-import numpy as np
-import geopandas as gpd
-import rioxarray
-from pyproj import CRS
-from .segmentmytif.src.segmentmytiff.main import make_predictions
-from dask.distributed import LocalCluster, Client
+from segmentmytif.main import read_input_and_labels_and_save_predictions
+from segmentmytif.features import FeatureType
+import logging
 
-FONTSIZE = 16
+# Turn off the logger
+logging.getLogger().setLevel(logging.CRITICAL)
 
+# Constants
+FONTSIZE = 16 # Font size for the labels
+LABEL_HEIGHT = 20 # Height of the labels
+WIDGET_WIDTH = 600 # Width of all the widgets
+WIDGET_HEIGHT = 25 # Height of all non-label widgets
 
 class DroneMLDialog(QtWidgets.QDialog):
     def __init__(self, parent=None):
```

Comment:
We have such nice descriptions of the options in the Readme now.  We should make them available in the UI itself somehow. We could even remove the description again from the readme to avoid having to maintain 2 documentations. Some options:
- As a tooltip. 
- under an information icon. 
- adding a link (with I or ? icon) that links to the Readme (simplest option)

Reply:
Created an issue #17